src/stores/global.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Global: this store is outside Antelope store because it is app specific and is responsible for handling the app global auxiliary state.
```

Comment:
I have mixed feelings about this global store because we're not defining what it is, just that it contains some globally accessible information.
My suggestion is to name it in a way that indicates we're using it to manage the app navigation state. What do you think?

Reply:
I don't have a strong opinion about it... but, it is exactly what you described, "some globally accessible information". In fact, we already have this store but using the old _Vuex_ for the Native wallet and you can see it is used for all kinds of global auxiliary information. I'm not saying is a good practice, just following the pattern, but we can change it if you want.

https://github.com/telosnetwork/telos-wallet/blob/177-wallet-%7C-receive-page/src/store/global/getters.js#L1-L7

Anyway, this is a detail that I would like to write a new issue to address it later when having time to do some refactoring, and in the midtime, we can discuss it and have more opinions on the matter.

Reply:
Sure!
I created [this issue](https://github.com/telosnetwork/telos-wallet/issues/271) to address this refactoring in the future. Currently, this solution touches several files and is present in two PRs (both receive and send pages use this code). So, it will be easy to refactor after it gets merged.